### PR TITLE
update readme to have more verbose instructions

### PR DESCRIPTION
**Pull request by justinmclark**
Edited directly in Github.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,23 +9,40 @@ using [Elasticsearch](https://www.elastic.co/).
 Increase the maximum number of virtual memory areas on your host
 system. On a Linux system this can be done using the command line
 
-    sudo sysctl -w vm.max_map_count=262144
+    sysctl -w vm.max_map_count=262144
```
Comment (balbirthomas) [inline README.md:12]: Are you sure about this ? I mean does changing system config as normal user always work ?

**Review by balbirthomas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (justinmclark) [reply README.md:12]: It does have to be as root, but for some reason I have an aversion to including `sudo` in tutorials. Maybe I should get over this.. I liked how the ES docs put it: "now run the following command as root": `sysctl -w vm.max_map_count=262144`

Thoughts?

https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html

Comment (dstathis) [reply README.md:12]: I also don't like when sudo is in docs. I you don't know when to use sudo, you need to better understand the possibly system breaking commands you are about to run.

**Review by dstathis — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (balbirthomas) [reply README.md:12]: Either is fine - explicitly state that the following command must be run as root or add `sudo` to the command prefix. Take your pick.

**Review by balbirthomas — APPROVED** [review]

**Comment by justinmclark** [comment]
@balbirthomas I decided to just go with `sudo` and get over my aversion. Thanks.

File: README.md [hunk 2]
```diff
@@ -13,19 +13,36 @@ system. On a Linux system this can be done using the command line
...
     charmcraft build
 
 ## Usage
 
-TODO: explain how to use the charm
+    juju deploy ./elasticsearch.charm
+
+To scale up:
+
+    juju add-units -n 2 elasticsearch
```
Comment (balbirthomas) [inline README.md:34]: `add-unit` or `add-units` ?

File: README.md [hunk 3]
```diff
@@ -13,19 +13,36 @@ system. On a Linux system this can be done using the command line
...
+
+> Note: When the total number of nodes in the cluster is 2, split brain is possible. If there are currently two nodes, be sure to use `juju add-units` to scale up to a functional HA cluster.
```
Comment (balbirthomas) [inline README.md:36]: Wonder if "split brain" should be linked to Elasticsearch documentation page explaining what it is and what the consequences are ?

**Review by balbirthomas — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (justinmclark) [reply README.md:34]: Ah good catch.

File: README.md (hunk 3 above)
Comment (justinmclark) [reply README.md:36]: I like this idea. Will add.

**Review by balbirthomas — APPROVED** [review]

**Pull request merged**